README.md

```diff
@@ -306,7 +306,25 @@ val df = spark.readStream.format("deltaSharing")
 [magpierre/delta-sharing](https://github.com/magpierre/delta-sharing/tree/cppdev/cpp/DeltaSharingClient)
 </td>
 <td>Released</td>
-<td>QeuryTableMetadata<br>QueryTableLatestSnapshot</td>
+<td>QueryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>R</td>
+<td>
+
+[zacdav-db/delta-sharing-r](https://github.com/zacdav-db/delta-sharing-r)
+</td>
+<td>Released</td>
+<td>QueryTableVersion<br>QueryTableMetadata<br>QueryTableLatestSnapshot</td>
+</tr>
+<tr>
+<td>Google Spreadsheet</td>
+<td>
+
+[TBD](TBD)
```

Comment:
@wchau What's the status and link?

Reply:
You can point to https://github.com/delta-incubator/delta-sharing-connectors/tree/main/google_workspace_add_on for now. We can label it as Beta.